plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -83,7 +83,7 @@
     version_added: 1.3.0
   clone:
     description:
-      - Name of VM to be cloned. If C(vmid) is setted, C(clone) can take arbitrary value but required for initiating the clone.
+      - Name of VM to be cloned. If C(vmid) is set, C(clone) can take an arbitrary value but is required for initiating the clone.
```

Comment:
While we're here:
```suggestion
      - Name of VM to be cloned. If I(vmid) is set, I(clone) can take an arbitrary value but is required for initiating the clone.
```
(https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation)